docs/modules/operation/pages/service-assurance/monitors/DNSResolutionMonitor.adoc

```diff
@@ -2,49 +2,57 @@
+The DNS resolution monitor tests if the node label of a {page-component-title} node can be resolved.
+This monitor uses the name resolver configuration from the poller configuration or from the operating system where {page-component-title} is running.
+Use it to test client behavior for a given host name.
+
+For example, create a node with the node label www.google.com and an IP interface.
+Assigning the DNS resolution monitor on the IP interface tests if www.google.com can be resolved using the DNS configuration defined by the poller.
+The response from the A record lookup can be any address; it is not verified with the IP address on the {page-component-title} IP interface where the monitor is assigned.
+
 This monitor implements <<service-assurance/monitors/introduction.adoc#ga-service-assurance-monitors-placeholder-substitution-parameters, placeholder substitution in parameter values>>.
 
-== Monitor Facts
+== Monitor facts
 
 [options="autowidth"]
 |===
 | Class Name | `org.opennms.netmgt.poller.monitors.DNSResolutionMonitor`
 | Remote Enabled | true
 |===
 
-== Configuration and Use
+== Configuration and use
 
 .Monitor specific parameters for the DNSResolutionMonitor
-[options="header, autowidth"]
+[options="header"]
+[cols="1,3,2"]
+
 |===
-| Parameter         | Description                                       | Required | Default value | Placeholder substitution
-| `resolution-type` | Type of record for the node label test. +
+| Parameter         | Description |               Default 
+| resolution-type| Type of record for the node label test. +
                       Allowed values +
-                      `v4` for _A records_, +
-                      `v6` for _AAAA record_, +
-                      `both` _A_ and _AAAA record_ must be available, +
-                      `either` _A_ or _AAAA record_ must be available.  | optional | `either` | No
-| `record-types`    | Alternate DNS record types to search for. +
-                      The comma separated list can contain `A`, +
-                      `AAAA`, `CNAME`, `NS`, `MX`, `PTR`, `SOA`, +
-                      `SRV`, or `TXT`.                                  | optional | `` | No
-| `lookup`          | Alternate DNS record to lookup                    | optional | The node label. | Yes
-| `nameserver`      | The DNS server to query for the records. +
+                      `v4` for A records, +
+                      `v6` for AAAA records, +
+                      `both` A and AAAA records must be available, +
+                      `either` A or AAAA records must be available.  | either 
+| record-types   | Alternate DNS record types to search for. +
+                      The comma-separated list can contain A, +
+                      AAAA, CNAME, NS, MX, PTR, SOA, +
+                      SRV, or TXT.                                  | n/a 
+| lookup 
```

Comment:
It seems like the `{}` is a pro commercial icon in font awesome 6.0+ which we can't use right now easily :/ Naive idea :) Instead of having the **{}** in a new line like here:
![Screenshot 2021-07-12 at 10 26 57](https://user-images.githubusercontent.com/1095181/125258492-b0c1e600-e2fe-11eb-836d-407fddc7ebe2.png)

we could use the Kbd button macro `kbd:[{}]` behind the parameter name like here:
![Screenshot 2021-07-12 at 10 16 06](https://user-images.githubusercontent.com/1095181/125258860-0ac2ab80-e2ff-11eb-90ae-94893306f05d.png)

Reply:
That works much better. Thanks!

Reply:
Why are the `{}` on those parameters?  I saw it on a few others but not sure what it is signifying.

Reply:
They indicate parameters that support placeholder substitution. The introduction to the monitors section notes this, but I think I should add that note with each table that uses that symbol.

Reply:
Ah. Yeah, as someone that's been using the system, I wasn't likely to re-visit the intro page. A footnote at the bottom of the table might be helpful to transition users.

Reply:
I'll do that when I go in to update the actual icons, and any other changes required.